starter-core/src/test/groovy/io/micronaut/starter/feature/aws/DynamoDbSpec.groovy

```diff
@@ -51,15 +51,12 @@ class DynamoDbSpec extends ApplicationContextSpec implements CommandOutputFixtur
                 .language(language)
                 .features([DynamoDb.NAME, GraalVM.FEATURE_NAME_GRAALVM])
                 .render()
-        String mapNotation = buildTool == BuildTool.GRADLE ? ':' : ' ='
         BuildTestVerifier verifier = BuildTestUtil.verifier(buildTool, language, template)
 
         then:
         verifier.hasDependency("io.micronaut.aws", "micronaut-aws-sdk-v2", Scope.COMPILE)
-        template.contains("""    implementation("software.amazon.awssdk:dynamodb") {
-                            |      exclude(group$mapNotation "software.amazon.awssdk", module$mapNotation "apache-client")
-                            |      exclude(group$mapNotation "software.amazon.awssdk", module$mapNotation "netty-nio-client")
-                            |    }""".stripMargin())
+        verifier.hasExclusion("software.amazon.awssdk", "apache-client")
+        verifier.hasExclusion("software.amazon.awssdk", "netty-nio-client")
```

Comment:
Here again, this isn't testing the same thing...

I think the new hasExclusion method is missing the concept of _where_ it's excluded from